inc/Engine/Optimization/DelayJS/Admin/Settings.php

```diff
@@ -217,7 +217,7 @@ public function option_update_3_7_6_1( $old_version ) {
 
 		$options = get_option( 'wp_rocket_settings', [] );
 
-		$options['delay_js_scripts'] = $this->defaults;
+		$options['delay_js_scripts'] = array_values( array_unique( array_merge( $this->defaults, $options['delay_js_scripts'] ) ) );
```

Comment:
Do we know with certainty that `$options['delay_js_scripts']` exists and it's an array?

Reply:
Since we're updating from a specific version where the array has been defined previously, it should be there. Of if it's not it  something else has gone (badly) wrong.